SpatialGDK/Documentation/known-issues.md

```diff
@@ -25,3 +25,4 @@ Known issue = any major user-facing bug or lack of user-facing feature that:
 | Generating schema for `UClass`es that are not explicitly loaded/referenced by the currently loaded level is not supported. For example, a hard coded path loaded at run-time will not be detected by the schema generator. | 2019-03-25 |  | Reference the class explicitly from an already loaded class (Level blueprint, GameMode etc). Alternatively, you can open the Blueprints for soft references before generating schema so that they are loaded into memory. |
 | Level streaming might affect performance in certain scenarios. | 2019-03-26 | [UNR-1163](https://improbableio.atlassian.net/browse/UNR-1163) | Select the **Query Based Interest Enabled** checkbox in the SpatialOS GDK for Unreal Runtime Settings to enable or disable level streaming. If the **Query Based Interest Enabled** checkbox is ticked, level streaming will be supported, but this might affect performance in certain scenarios and is currently being investigated. |
 | Changing some SpatialOS settings after launching a local deployment will invalidate the currently launched deployment (e.g. Changing the number of servers, or toggling using Query Based Interest), as these settings modify the launch configuration. | 2019-03-26 | [UNR-1190](https://improbableio.atlassian.net/browse/UNR-1190) | Manually stop and start your deployment after you make any changes that alter the launch configuration. |
+| GDK ignores the bHidden specifier for actors | 2019-04-16 | [UNR-1182](https://improbableio.atlassian.net/browse/UNR-1182) |  Use UCLASS(SpatialType = (Singleton, ServerOnly)) |  
```

Comment:
I'm not sure this workaround really solves the problem. bHidden can be toggled dynamically per actor, and marking things ServerOnly is static per class.

Reply:
I agree with you, should this workaround not be mentioned then?

Reply:
Discussed offline, switch workaround to use bReplicates instead.

Reply:
Updated the workaround.